app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterClaimController.scala

```diff
@@ -86,10 +86,17 @@ class EnterClaimController @Inject() (
           case Some(claim) =>
             fillingOutClaim.draftClaim.fold(_.isMrnFlow) match {
               case true  =>
-                val form = mrnClaimAmountForm(claim.paidAmount).fill(ClaimAmount(claim.claimAmount))
+                val emptyForm = mrnClaimAmountForm(claim.paidAmount)
```

Comment:
how do you know this is empty? is `claim.paidAmount` not filled at this point?

Reply:
Because I'm just creating it right in line 89. So it must be empty.
(I call the .fill method on the next line)